webui/src/app/components/env-chip/env-chip.component.ts

```diff
@@ -2,8 +2,9 @@ import { Component, Input, OnInit } from '@angular/core';
 import { EnvironmentsStoreService } from '../../stores/environments-store.service';
 import { EnvironmentBean } from '../../models/commons/applications-bean';
 import { Store } from '@ngrx/store';
-import { ISubscription } from 'rxjs/Subscription';
+import { SubscriptionLike as ISubscription } from 'rxjs';
 import { AutoUnsubscribe } from '../../shared/decorator/autoUnsubscribe';
+import { Observable } from 'rxjs/internal/Observable';
```

Comment:
Sounds very weird to rely on someone else' internals! Auto-import typo for `rxjs/Observable`?